### ISSUE TITLE
Panic when importing cdrs via ers

### ISSUE THREAD

**Issue by miha-** [cgrates/cgrates#2123]
Hi

when importing cdrs via ers (csv file) i get this error.

Let me know what else do you need so that you will be able to debug this.

thank you!
miha

cgr-console -version
CGRateS@v0.11.0~dev-20200408183212-e7cf6b2687e1

pr 29 10:05:04 debian cgr-engine[12362]: #015
Apr 29 10:05:04 debian cgr-engine[12362]: #033[35m(/root/work/src/github.com/cgrates/cgrates/engine/storage_sql.go:879)#033[0m
Apr 29 10:05:04 debian cgr-engine[12362]: #033[33m[2020-04-29 10:05:04]#033[0m #033[31;1m Error 1062: Duplicate entry '20ab029ed95a2ac421f6f7459d527f52547560a0-*default' for key 'cdrrun' #033[0m
Apr 29 10:05:04 debian cgr-engine[12362]: panic: runtime error: invalid memory address or nil pointer dereference
Apr 29 10:05:04 debian cgr-engine[12362]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xfcbc1a]
Apr 29 10:05:04 debian cgr-engine[12362]: goroutine 197897 [running]:
Apr 29 10:05:04 debian cgr-engine[12362]: github.com/cgrates/cgrates/engine.(*CallDescriptor).refundIncrements(0xc0006a8dc0, 0x0, 0x0, 0x0)
Apr 29 10:05:04 debian cgr-engine[12362]: #011/root/work/src/github.com/cgrates/cgrates/engine/calldesc.go:886 +0xfa
Apr 29 10:05:04 debian cgr-engine[12362]: github.com/cgrates/cgrates/engine.(*CallDescriptor).RefundIncrements.func1(0xc000078190, 0xc000766468, 0xc000766470, 0xc000652280)
Apr 29 10:05:04 debian cgr-engine[12362]: #011/root/work/src/github.com/cgrates/cgrates/engine/calldesc.go:938 +0x33
Apr 29 10:05:04 debian cgr-engine[12362]: github.com/cgrates/cgrates/guardian.(*GuardianLocker).Guard.func1(0xc00617e940, 0xc006f995c0, 0xc006f99620)
Apr 29 10:05:04 debian cgr-engine[12362]: #011/root/work/src/github.com/cgrates/cgrates/guardian/guardian.go:138 +0x38
Apr 29 10:05:04 debian cgr-engine[12362]: created by github.com/cgrates/cgrates/guardian.(*GuardianLocker).Guard
Apr 29 10:05:04 debian cgr-engine[12362]: #011/root/work/src/github.com/cgrates/cgrates/guardian/guardian.go:136 +0x11b
Apr 29 10:05:05 debian CGRateS <6215b44> [13211]: <CoreS> starting version <CGRateS@v0.11.0~dev-20200408183212-e7cf6b2687e1><go1.14.2>

**Comment by TeoV** [comment]
Hi @miha-,

Can you please attach the config file, a sample of data and to reproduce this panic?
Also please attach the ngrep between SessionS, CDRs and RALs.

Thanks,
TeoV

**Comment by miha-** [comment]
@TeoV I will try to get everything over weekend and then I will get back to you.

br
miha

**Comment by miha-** [comment]
@TeoV i tried this to reproduce again. The main issue was with cgird as i was composing it and i used some values in which there were spaces. After removing spaces everything seems to be well.

I guess you can close this ticket.
br
miha

**Comment by TeoV** [comment]
Hi @miha-,

Glad to hear that it worked out on its own.

Thanks,
TeoV

**Comment by miha-** [comment]
@TeoV Hi!

I think there is another issue with this.

Jul  8 10:39:10 debian cgr-engine[18520]: #015
Jul  8 10:39:10 debian cgr-engine[18520]: #033[35m(/root/work/src/github.com/cgrates/cgrates/engine/storage_sql.go:879)#033[0m
Jul  8 10:39:10 debian cgr-engine[18520]: #033[33m[2020-07-08 10:39:10]#033[0m #033[31;1m Error 1062: Duplicate entry '1b7e81bba6708c3cfef4bee906d34e60dc6b034f-*raw' for key 'cdrrun' #033[0m
Jul  8 10:39:10 debian cgr-engine[18520]: panic: runtime error: invalid memory address or nil pointer dereference
Jul  8 10:39:10 debian cgr-engine[18520]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xfcbc1a]
Jul  8 10:39:10 debian cgr-engine[18520]: goroutine 10836 [running]:
Jul  8 10:39:10 debian cgr-engine[18520]: github.com/cgrates/cgrates/engine.(*CallDescriptor).refundIncrements(0xc00046cc80, 0x0, 0x0, 0x0)
Jul  8 10:39:10 debian cgr-engine[18520]: #011/root/work/src/github.com/cgrates/cgrates/engine/calldesc.go:886 +0xfa
Jul  8 10:39:10 debian cgr-engine[18520]: github.com/cgrates/cgrates/engine.(*CallDescriptor).RefundIncrements.func1(0xc000100190, 0xc0002a22f0, 0xc0002a2300, 0xc000706780)
Jul  8 10:39:10 debian cgr-engine[18520]: #011/root/work/src/github.com/cgrates/cgrates/engine/calldesc.go:938 +0x33
Jul  8 10:39:10 debian cgr-engine[18520]: github.com/cgrates/cgrates/guardian.(*GuardianLocker).Guard.func1(0xc000e9dd20, 0xc00247af60, 0xc00247b020)
Jul  8 10:39:10 debian cgr-engine[18520]: #011/root/work/src/github.com/cgrates/cgrates/guardian/guardian.go:138 +0x38
Jul  8 10:39:10 debian cgr-engine[18520]: created by github.com/cgrates/cgrates/guardian.(*GuardianLocker).Guard
Jul  8 10:39:10 debian cgr-engine[18520]: #011/root/work/src/github.com/cgrates/cgrates/guardian/guardian.go:136 +0x11b
Jul  8 10:39:11 debian CGRateS <df08a2c> [18749]: <CoreS> starting version <CGRateS@v0.11.0~dev-20200408183212-e7cf6b2687e1><go1.14.2>

Basically I am trying to find out what exactly triggers this but I am unable to.

[trace.txt](https://github.com/cgrates/cgrates/files/4889555/trace.txt)

If you need anything else please let me know.

thank you
miha
